### ISSUE TITLE
Prioritized left turns wait for right turns

### ISSUE THREAD

**Issue by lola-notraffic** [eclipse/sumo#11728]
Hello.
In tested simulation left turns defined as prioritized above right turns that must enter the same lane:
in the video below:
Left turns vehicles give priority to straight coming vehicles. Left turn vehicles get priority above right turn vehicles, but they wait a lot of time before they turn left in front of right vehicles (you may see the video). 

Video in the link, permissions granted:
https://drive.google.com/drive/folders/1358oITtrCDt6wPItnat7Pua_RhLLcg7O?usp=sharing

But from what we saw: vehicles on the junction don't wait so much time each
NET file definitions of priorities:
![image](https://user-images.githubusercontent.com/95179338/192782746-e9cf94fc-6f0e-4ce6-be5d-b0c86174cdea.png)

**SUMO-version: 13.0**

**operating system: ubuntu 20.04**

**Comment by namdre** [comment]
Modifying the `<request>` elements in the network file is likely insufficient since there are additional right-of-way related settings encoded in the internal junctions and the link states (https://sumo.dlr.de/docs/FAQ.html#i_made_changes_to_the_netxml-file_but_it_did_not_work_as_expected_why)

You could instead try to define the desired right of way rules by loading `<prohibitions>`. See https://sumo.dlr.de/docs/Networks/PlainXML.html#setting_connection_priorities

**Comment by lola-notraffic** [comment]
I'll check this one. Thank you!

**Comment by lola-notraffic** [comment]
I have added prohibition into net file under connections too.
![image](https://user-images.githubusercontent.com/95179338/192993272-ad613318-3058-4cda-8ea3-c9eaad337da5.png)
But left turns still wait for right turns as in previous video.

**Comment by lola-notraffic** [comment]
It seems like a bug maybe:
If map looks like this: (only right turn from the lane) - left vehicles don't wait:
![image](https://user-images.githubusercontent.com/95179338/193002285-eb135ab4-0e6c-467a-aad9-ed75f18ce932.png)
But if from the same right turn lane there is a straight direction too opposite left turn vehicles will wait:
![image](https://user-images.githubusercontent.com/95179338/193002744-f6891d3e-2a79-492e-b33d-9c6315b98f7a.png)

**Comment by lola-notraffic** [comment]
I will try this. Thank you!

**Comment by namdre** [comment]
Closing for lack of feedback. Add a comment if the issue isn't resolved.

**Comment by lola-notraffic** [comment]
Hi again. 
I ran the patch con file with netconvert command to create updated network, but the behavior of the left turns stayed the same. they still wait for right turns that entering the same lane. I verified that priorities were defined correctly too.
here is how configuration looks:
![Selection_070](https://user-images.githubusercontent.com/95179338/216818184-5222bbe3-8491-4050-8796-3cc797119424.png)

Thank you in advance.

SUMO-version: 13.0
operating system: ubuntu 20.04
